config/config.go

```diff
@@ -68,6 +68,9 @@ type Configuration struct {
 
 	GenesisEth1Block uint64 `mapstructure:"genesis_eth1_block"`
 	AppID            string `mapstructure:"app_id"`
+
+	BidAmount  uint64 `mapstructure:"bid_amount"`
```

Comment:
I've been looking for the right word for this. What do you think [WillingnessToPay](https://en.wikipedia.org/wiki/Willingness_to_pay)?

Reply:
Sounds good to me!